### PR TITLE
Home page card plus header/ footer with some styling

### PR DESCRIPTION
**Pull request by RiveraDarwin**

### CODE REVIEW

File: app/javascript/components/App.js
Comment (WalkerTexas-coder) [inline app/javascript/components/App.js]: Do you need all of the props for Devise login and logout being brought into your footer?

**Review by WalkerTexas-coder — COMMENTED** [review]
Got a question before approval, check it out below...

File: app/javascript/components/App.js
Comment (RiveraDarwin) [reply app/javascript/components/App.js]: I wanted to test something out after I had an error and forgot to remove it. Thank you for catching it.

**Comment by RiveraDarwin** [comment]
change pushed to branch home-page

**Review by SunkissedQueen — APPROVED** [review]
Looks good from this angle

**Pull request merged**
